spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,63 @@
+
+
+
+describe 'User Stories' do
+  let(:takeaway) { Takeaway.new }
+  let(:menu) { Menu.new}
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+
+  context '#menu' do
+
+
+    it 'displays a menu with a list of dishes with prices' do
+      expect(menu.dishes).to eq ({
+        :salad => 12,
+        :burger => 14,
+        :fajita => 13,
+        :pasta => 8,
+        :brownies => 9
+      })
+    end
+
+    it 'finds a specific dish and returns the price' do
+      expect(menu.dishes[:burger]).to eq 14
+    end
+  end
+
+  # As a customer
+  # So that I can order the meal I want
+  # I would like to be able to select some number of several available dishes
+
+  it 'selects items from the menu which is stored as an order' do
+    expect(takeaway.select(:salad, 2)).to eq "You have added 2 salad(s) to your basket"
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)